aiida/cmdline/commands/cmd_status.py

```diff
@@ -82,10 +82,9 @@ def verdi_status():
         with Capturing(capture_stderr=True):
 
             postgres = Postgres.from_profile(profile)
-            pg_connected = postgres.try_connect()
 
-        dbinfo = postgres.get_dbinfo()
-        if pg_connected:
+        dbinfo = postgres.dbinfo.copy()
```

Comment:
Why do we need a copy here? Will the `postgres.is_connected` call modify it?

Reply:
It was just the safe replacement. It's not needed here.